modules/ROOT/pages/create-community.adoc

```diff
@@ -106,7 +106,7 @@ After you create a community, you must create different user profiles for your c
 Configure the settings with one of these options:
 
 * Configure the member and guest user settings automatically using the default settings specified by the *Fix* buttons in the xref:diagnostics.adoc#user-access-tab[API Community Manager diagnostics user access tab].
-* Configure the member and guest user settings manually using custom settings applied in a series of steps.
+* Configure the member and guest user settings manually using custom settings applied in the following steps.
```

Comment:
So, the problem I see with referring to "the following steps" when the steps are not directly following this line is that, should the topics be reused/rearranged, we have a broken reference. We want to avoid creating topic dependencies....

Reply:
The steps do directly follow this line, in the subsections "Configure Member User Profile Permissions" and "Configure Guest User Profile Permissions".